homeassistant/components/local_ip/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "title": "Local IP Address",
+    "step": {
+      "user": {
+        "title": "Local IP Address",
+        "data": {
+          "name": "Sensor Name"
+        }
+      }
+    },
+    "error": {
+      "unknown": "Unexpected error"
+    },
+    "abort": {
+      "already_configured": "Integration is already configured"
```

Comment:
This string isn't used.

Reply:
I thought it was used in the base class somewhere....  It was part of the default scaffolding code, but unused by the default scaffolding code, too...

Reply:
FYI, ended up actually using the abort string, but got rid of the errors (per the try/catch edit)